spec/controllers/procedures_controller_spec.rb

```diff
@@ -5,7 +5,7 @@
     it 'gives a list of procedures and their average sensation and satisfaction' do
       procedures = create_list(:procedure, 2)
       get :index
-      expect(assigns(:procedures)).to match_array(procedures)
+      expect(assigns(:procedures)).to contain_exactly(*procedures)
```

Comment:
Hmm, I would expect `match_array` to be better here since it's unordered. Why the switch?

Reply:
Weird, so when I originally did this (pre-rebase) it was `to_eq`. No idea what happened, I'll revert this.